css-conditional-5/Overview.bs

```diff
@@ -336,4 +1241,40 @@ Additions since Level 4</h3>
 		<li>Added ''@when'' and ''@else''.
 		<li>Extended [=supports queries=] to express font capabilities
 			via ''font-tech()'' and ''font-format()''.
+		<li>Moved Container Queries from [[CSS-CONTAIN-3] to this specification.
+			(See also the [[CSS-CONTAIN-3#changes]] for more information
+			on the evolution of this feature.)
 	</ul>
+
+<h2 class="no-num" id="acknowledgments">Acknowledgments</h2>
+
+	Comments and previous work from
+	Adam Argyle,
+	Amelia Bellamy-Royds,
+	Anders Hartvoll Ruud,
+	Brian Kardell,
+	Chris Coyier,
+	Christopher Kirk-Nielsen,
+	David Herron,
+	Elika J. Etemad (fantasai),
+	Eric Portis,
+	Ethan Marcotte,
+	Geoff Graham,
```

Comment:
Also, I suspect @frivoal should probably be added to the Acknowledgments section (as a now-non-editor).

Reply:
done